lldb/source/Commands/CommandObjectDisassemble.cpp

```diff
@@ -351,8 +351,8 @@ CommandObjectDisassemble::GetNameRanges(CommandReturnObject &result) {
 
   // Find functions matching the given name.
   SymbolContextList sc_list;
-  GetTarget().GetImages().FindFunctions(name, eFunctionNameTypeAuto,
-                                        function_options, sc_list);
+  GetTarget().GetImages().FindFunctions(
+      name, eFunctionNameTypeAuto, function_options, SymbolContext(), sc_list);
```

Comment:
If we use the default parameter as suggested above, all changes in this file go away.